tests/chainer_tests/functions_tests/noise_tests/test_dropout.py

```diff
@@ -183,7 +184,8 @@ def test_cpu_ideep(self):
     def test_gpu(self):
         self.x = cuda.to_gpu(self.x)
         self.mask = cuda.to_gpu(self.mask)
-        self._check()
+        with chainer.using_config('use_cudnn', 'never'):
+            self._check()
```

Comment:
How about adding a new test for `use_cudnn='always'` (as `test_gpu_cudnn`)?
Perhaps it's better to use `inject_backend_tests`.

Reply:
I added tests, but it is possibly implemented in a wrong way.

Reply:
Thanks!